README.md

```diff
@@ -7,6 +7,10 @@
 ## Overview
 This repo contains utils for implementing a [slam service](https://github.com/viamrobotics/rdk/blob/main/services/slam/slam.go) in Golang.
 
+Available implementations:
+- [Cartographer](https://github.com/viamrobotics/viam-cartographer)
+- [ORB_SLAM3](https://github.com/viamrobotics/viam-orb-slam3)
+
```

Comment:
@zaporter-work, we decided in one of the meetings not to link to our modules. I'll remove this again.